src/window/gui/Gui.cpp

```diff
@@ -69,7 +69,8 @@ Gui::Gui(std::vector<std::shared_ptr<GuiWindow>> guiWindows) : mNeedsConsoleVari
     }
 
     if (GetGuiWindow("Input Editor") == nullptr) {
-        AddGuiWindow(std::make_shared<InputEditorWindow>(CVAR_CONTROLLER_CONFIGURATION_WINDOW_OPEN, "Input Editor"));
+        AddGuiWindow(std::make_shared<InputEditorWindow>(CVAR_CONTROLLER_CONFIGURATION_WINDOW_OPEN, "Input Editor",
+                                                         ImVec2(520, 600)));
```

Comment:
the input editor window didn't have a hardcoded size before, does this make it super wide? also the window itself was titled "Controller Configuration" - does this make it "Input Editor" now?

Reply:
Yeah, considering the changes I made to how the name displays, that did change it. I think it then is tied to the name used for lookup here. Should I just change it to Controller Configuration here?

Reply:
I prefer controller config as a name but if others have strong feelings I'm fine with any name

Reply:
Actually, considering we do support keyboard (and I to want to support mouse eventually), perhaps Input Editor is a better window title?

Reply:
I did remove the initial window size, though. Letting it autosize itself makes more sense.